File: backend/ria_api/client_create.py
def client_create():
    # create a client data structure with default values
    client = {
        'p1Name': 'Bob',
        'p1Sex': 'M',
        'p1Age': 67,
        'p2Name': 'Sue',
        'p2Sex': 'F',
        'p2Age': 65,
        'Year': 2015,
        'nScenarios': 100000,
        'budget': 1000000,
        # figure size in pixels: width, height
        # set to [] to use full screen
        'figureSize': [1500, 900]
    }
    return client

File: backend/ria_ml/client_create.m
function client = client_create()
  % create a client data structure with default values
    client.p1Name = 'Bob';
    client.p1Sex =  'M';
    client.p1Age =  67;
    client.p2Name = 'Sue';
    client.p2Sex = 'F';
    client.p2Age =  65;
    client.Year = 2015;
    client.nScenarios = 100000;
    client.budget = 1000000;
    % figure size in pixels: width, height
    %    set to [ ] to use full screen
      client.figureSize = [1500  900];
end
